static/js/src/advantage/react/components/Subscriptions/SubscriptionCancel/SubscriptionCancel.test.tsx

```diff
@@ -8,7 +8,7 @@ describe("SubscriptionCancel", () => {
   it("handles closing the modal", () => {
     const onClose = jest.fn();
     const wrapper = shallow(<SubscriptionCancel onClose={onClose} />);
-    wrapper.find(Modal).invoke("close")();
+    wrapper.find(Modal).invoke("close")!();
```

Comment:
What is that syntax?

Reply:
This is a [non-null assertion](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator). Because `close` is an optional prop we need to tell Typescript to call the function only if it exists (which it will in this case if we've passed the `onClose` function correctly).

Reply:
Oh interesting, thanks!